base/FileIOShared.F90

```diff
@@ -599,11 +599,11 @@ subroutine ArrDescrCreateWriterComm(arrdes, full_comm, num_writers, rc)
        integer, intent(in) :: num_writers
        integer, optional, intent(out) :: rc
 
-       integer :: status, nx, ny, color, ny_by_writers, myid, j, writer_rank
+       integer :: status, nx, ny, color, ny_by_writers, myid, j
 
        nx = size(arrdes%i1)
        ny = size(arrdes%j1)
-       _ASSERT(num_writers <= ny,'num writers must be less or equal to than NY')
+       _ASSERT(num_writers < ny,'num writers must be less than NY')
```

Comment:
I'm pretty sure this has to be `<=` here and not just `<`. I believe the SCM will fall apart otherwise.

Reply:
yep I fixed this